tests/foreman/cli/test_host.py

```diff
@@ -161,17 +162,15 @@ def test_positive_create_with_name(self):
             with self.subTest(name):
                 host = entities.Host()
                 host.create_missing()
-                result = Host.create({
+                result = make_host({
                     u'architecture-id': host.architecture.id,
                     u'domain-id': host.domain.id,
                     u'environment-id': host.environment.id,
-                    # pylint:disable=no-member
```

Comment:
Try executing `pylint tests/foreman/cli/test_host.py` from robottelo dir (if you have requirements-optional installed).